### PR TITLE
Update names for some views in Objective-C

### PR DESCRIPTION
**Pull request by martinmitrevski**
### 🎯 Goal

Prevent name collisions in Objective-C based projects.

### ☑️ Contributor Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [ ] This change follows zero ⚠️ policy (required)
- [ ] This change should be manually QAed
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [ ] Affected documentation updated (docusaurus, tutorial, CMS)

### 🎁 Meme

Alt Text

### CODE REVIEW

**Review by nuno-vieira — APPROVED** [review]
LGTM!

File: Sources/StreamChatUI/CommonViews/PlayerView.swift [hunk 1]
```diff
@@ -7,6 +7,7 @@ import StreamChat
 import UIKit
 
 /// A view that shows a playing video content.
+@objc(StreamPlayerView)
```
Comment (nuno-vieira) [inline Sources/StreamChatUI/CommonViews/PlayerView.swift:10]: @martinmitrevski We forgot the changelog

Comment (martinmitrevski) [reply Sources/StreamChatUI/CommonViews/PlayerView.swift:10]: do we need it here? I can raise another PR if you think so.

Comment (nuno-vieira) [reply Sources/StreamChatUI/CommonViews/PlayerView.swift:10]: I think it makes sense, it fixed a problem for a customer, so more customers can have the same issue

**Pull request merged**